server/api/core/email.js

```diff
@@ -0,0 +1,59 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Job } from "meteor/vsivsi:job-collection";
+import { Jobs, Packages, Templates } from "/lib/collections";
```

Comment:
We seem to be doing it here: https://github.com/reactioncommerce/reaction/blob/b7613521d4eb0a4cf07cf98a8853b8fb506fe499/server/startup/i18n.js#L3-L3